composer.json

```diff
@@ -9,7 +9,7 @@
     }],
     "require": {
         "silverstripe/framework": "^4.0@dev",
-        "colymba/gridfield-bulk-editing-tools": "~2.1"
+        "colymba/gridfield-bulk-editing-tools": "^3.0@dev"
```

Comment:
Without this it pulls 2-x- goes along with the minimum stability flag - they were both needed when I was testing it on my fork. They're only temporary until we have an alpha tag released anyway